docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_keyvault_extension/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+type: docs
+title: "Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions"
+linkTitle: "Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions"
+weight: 3
+description: >
+---
+
+## Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions
+
+The following README will guide you on how to enable [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/overview) for a Cluster API that is projected as an Azure Arc connected cluster.
+
+In this guide, you will hook the Cluster API to Azure Key Vault by deploying the [Azure Key Vault extension](https://docs.microsoft.com/en-us/azure/aks/csi-secrets-store-driver) and a sample app on your Kubernetes cluster in order to integrate Azure Key Vault as a secrets store with a Kubernetes cluster via a CSI volume. This will also include deploying a Key Vault and sample secret on your Azure subscription.
+
+> **Note: This guide assumes you already deployed a Cluster API and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion using a [Shell script](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/cluster_api/capi_azure/).**
+
+Kubernetes extensions are add-ons for Kubernetes clusters. The extensions feature on Azure Arc-enabled Kubernetes clusters enables usage of Azure Resource Manager based APIs, CLI and portal UX for deployment of extension components (Helm charts in initial release) and will also provide lifecycle management capabilities such as auto/manual extension version upgrades for the extensions.
+
+## Prerequisites
+
+* Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
```

Comment:
is this used for the shell script only? should we have users download the script only instead of cloning the entire repo?

Reply:
This is a standard instruction for all jumpstart scenarios. I would prefer to keep this rather than downloading individual scripts. @Likamrat for any other input.

Reply:
+1 @dkirby-ms